client/src/assets/styles/constants/colors.css

```
@@ -0,0 +1,28 @@
+:root {
+  /* Colors */
+
+  --clr-green-400: #2a9d8f;
+  --clr-green-500: #264653;
+
+  --clr-yellow-400: #e9c46a;
+
+  --clr-white-400: #fff;
+  --clr-white-500: #f2e9e4;
+
+  --clr-blue-800: #22223b;
+
+  --clr-primary: var(--clr-green-400);

```

мб обойтись без дублирования констант? и просто везде писать 
```suggestion
  --clr-primary: #2a9d8f;

```